src/global/mfa.js

```diff
@@ -24,7 +24,7 @@ export const retrieve = async () => {
 
   mfa.totp = Object.assign({}, all.find(m => m.type === 'totp'))
   mfa.u2f = Object.assign({}, all.find(m => m.type === 'u2f'))
-  mfa.webauthn = Object.assign({}, all.find(m => m.type === 'webauthn'))
+  mfa.keys = Object.assign({}, all.find(m => m.type === 'webauthn'))
```

Comment:
I think [`find()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) only returns the first matching record, not all of the matching records (like `filter()` does). Is that the desired behavior?

Reply:
@hobbitronics Just following up, is `find()` really the appropriate method to use here? What kind of data is `mfa.keys` (aka. Object vs Array)?

Reply:
Okay, I see that mfa.keys is an object, not an array:
```json
{
  "totp": {},
  "u2f": {},
  "keys": {
    "id": 2,
    "type": "webauthn",
    "label": "Security Key",
    "created_utc": "2022-12-15T19:42:12Z",
    "last_used_utc": "2022-12-15T19:50:21Z",
    "data": [
      {
        "id": 3,
        "label": "SIL Mac TouchID",
        "last_used_utc": null,
        "created_utc": "2022-12-15 19:42:15"
      },
      {
        "id": 4,
        "label": "Blue Yubikey2",
        "last_used_utc": null,
        "created_utc": "2022-12-15 19:42:37"
      },
      {
        "id": 5,
        "label": "Personal smartphone",
        "last_used_utc": null,
        "created_utc": "2022-12-15 19:43:44"
      }
    ]
  },
  "backup": {},
  "numVerified": 3
}
```

Given that, it seems like `keys` is a bit misleading, if we want to change that back to `webauthn` in a future PR. Just fwiw.